jose/test/src/black/door/jose/jwt/CirceJsonJwtSpec.scala

```diff
@@ -0,0 +1,22 @@
+package black.door.jose.jwt
+
+import black.door.jose.Mapper
+import black.door.jose.json.circe.JsonSupport
+import io.circe.{Decoder, Encoder}
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+
+class CirceJsonJwtSpec extends JwtSpec with JsonSupport {
```

Comment:
Just plain `CirceJwtSpec` is fine. I only appended added the json word to the play tests to distinguish them from any future play framework integrations that may exist.